ext/bg/js/handlebars.js

```diff
@@ -75,6 +75,18 @@ function handlebarsMultiLine(options) {
     return options.fn(this).split('\n').join('<br>');
 }
 
+function handlebarsTermFrequencyColor(options) {
+    const termFrequency = options.fn(this);
+
+    if (termFrequency === 'popular') {
```

Comment:
Probably better to do this with classes. Having to update `js` files to change color scheme is odd.